frontend/src/employee-frontend/components/person-profile/FosterChildren.tsx

```diff
@@ -148,41 +150,77 @@ export default React.memo(function FosterChildren({
                 fosterChildren,
                 [({ child }) => child.lastName, ({ child }) => child.firstName],
                 ['asc', 'asc']
-              ).map(({ relationshipId, child, validDuring }) => (
-                <Tr
-                  key={relationshipId}
-                  data-qa={`foster-child-row-${child.id}`}
-                >
-                  <NameTd data-qa="name">
-                    <Link to={`/child-information/${child.id}`}>
-                      {child.firstName} {child.lastName}
-                    </Link>
-                  </NameTd>
-                  <Td>{child.socialSecurityNumber}</Td>
-                  <Td>
-                    {LocalDate.todayInHelsinkiTz().differenceInYears(
-                      child.dateOfBirth
-                    )}
-                  </Td>
-                  <Td data-qa="start">{validDuring.start.format()}</Td>
-                  <Td data-qa="end">{validDuring.end?.format() ?? ''}</Td>
-                  <Td>
-                    <Toolbar
-                      disableAll={!!uiMode}
-                      onEdit={() => startEditing(relationshipId, validDuring)}
-                      onDelete={() => startDeleting(relationshipId)}
-                      editable={true}
-                      deletable={true}
-                      dateRange={{
-                        startDate: validDuring.start,
-                        endDate: validDuring.end
-                      }}
-                      dataQaEdit="edit"
-                      dataQaDelete="delete"
-                    />
-                  </Td>
-                </Tr>
-              ))}
+              ).map(
+                ({
+                  relationshipId,
+                  child,
+                  creationModificationMetadata,
+                  validDuring
+                }) => {
+                  const modifiedAt =
+                    creationModificationMetadata.modifiedAt ||
+                    creationModificationMetadata.createdAt
+                  const modifiedByName =
+                    creationModificationMetadata.modifiedByName ||
+                    creationModificationMetadata.createdByName
```

Comment:
This might be unnecessary. If I understand the backend code correctly, modifier is never null but essentially defaults to the creator already there. Would it be better to actually have it as null in the db if the foster parent has been created but never modified, that I don't know.

Reply:
I'm probably going to get rid of the created_by field entirely and just have modified_at and modified_by (plus the fields set by triggers), since that's ultimately what was asked for and matches how things work elsewhere. Apparently the setup in these files is old and should probably be slowly replaced, which I didn't realize at first.

Reply:
I think keeping the created_at/created_by at least on the database level is good because if we later want to display or even just. manually check the original creator we can't do that if we haven't been storing the data. 

And yeah, most test files are full of legacy ways 😅